src/server/grpc_service.rs

```diff
@@ -621,6 +628,80 @@ impl<T: RaftStoreRouter + 'static> tikvpb_grpc::Tikv for Service<T> {
         ctx.spawn(future);
     }
 
+    fn raw_m_get(&self,
```

Comment:
It's generated code. If we want `raw_mget` in rust, we would have to change kvproto/golang from `RawMGet` to `RawMget`. What's your opinion?